tmpfiles.d/tmp.conf

```diff
@@ -8,8 +8,8 @@
 # See tmpfiles.d(5) for details
 
 # Clear tmp directories separately, to make them easier to override
-D /tmp 1777 root root -
-#d /var/tmp 1777 root root 30d
+d /tmp 1777 root root 10d
+d /var/tmp 1777 root root 30d
```

Comment:
Yeah, on my system I'd do something like 5 days and 10 days or something. I can't see how 30 day old files in /var/tmp could be useful.